assets/dev/js/editor/elements/widgets/base.js

```diff
@@ -0,0 +1,14 @@
+import ElementBase from 'elementor-elements/elements/base';
+
+export default class WidgetBase extends ElementBase {
+	/**
+	 * @return {string}
+	 */
+	getWidgetType() {
+		elementorModules.ForceMethodImplementation();
+	}
```

Comment:
Who is using that class? Now every widget will have to extend it?

Reply:
Only those who need custom Model,View...